docs/how-to/monitor/update-metrics.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Update metrics
+description: Update metrics manually if you're using a custom dashboard.
+sidebar_position: 3
+---
+
+# Update metrics manually
+
+If you are using a custom dashboard to monitor Teku metrics, you might need to update the metrics manually when breaking changes are introduced.
+
+This release of Teku has introduced breaking changes to metrics names. Every counter metric that did not have the suffix `_total` has incorporated the suffix. 
```

Comment:
Instead of `this release`, should we say the specific version?

Reply:
This is exactly what I changed in my last commit because we're can't ensure these docs will be released with 24.7.0 or 24.8.0 and the docs are already versioned. But I'm not strong opinionated on this.

Reply:
@gfukushima Should we hold this PR for publication until the release of 24.7.0 or 24.8.0?

Reply:
I think this is good to go as it is, worst case scenario I think we can edit this page to add the release number